integrations/langchain4j/providers/cohere/src/main/java/io/helidon/integrations/langchain4j/providers/cohere/CohereLc4jProvider.java

```diff
@@ -17,15 +17,11 @@
 package io.helidon.integrations.langchain4j.providers.cohere;
 
 import io.helidon.common.Weighted;
-import io.helidon.integrations.langchain4j.Ai;
-import io.helidon.service.registry.Qualifier;
+import io.helidon.integrations.langchain4j.AiProvider;
 
-final class Cohere {
-    static final String COHERE = "cohere";
-    static final String SCORING_MODEL = COHERE + "." + Ai.SCORING_MODEL_NAME;
-    static final Qualifier COHERE_QUALIFIER = Qualifier.createNamed(COHERE);
-    static final double WEIGHT = Weighted.DEFAULT_WEIGHT - 20;
+import dev.langchain4j.model.cohere.CohereEmbeddingModel;
 
-    private Cohere() {
-    }
+@AiProvider.ModelConfig(value = CohereEmbeddingModel.class,
+                        weight = Weighted.DEFAULT_WEIGHT - 20)
+interface CohereLc4jProvider {
```

Comment:
We already have a `Weight` annotation - do we need to have it as part of `ModelConfig`? If more than one `ModelConfig` can exist on one provider with different weights, it is OK.